tests_end_to_end/test_e2e_demisto_sdk_flow_playbook.py

```diff
@@ -0,0 +1,113 @@
+
+    unique_id = random.randint(1, 1000)
+    pack_name = 'foo_' + str(unique_id)
+    pack = repo.create_pack(name=pack_name)
+    playbook_name = 'pb_' + pack_name
+    playbook: Playbook = pack.create_playbook(name=playbook_name)
+    playbook.create_default_playbook(name=playbook_name)
+    assert path.exists(f'{tmpdir}/Packs/{pack_name}/Playbooks/{playbook_name}.yml')
+
+    print(f'Trying to upload playbook from {tmpdir}/Packs/{pack_name}/Playbooks/{playbook_name}.yml')
+    Uploader(input=f'{tmpdir}/Packs/{pack_name}/Playbooks/{playbook_name}.yml', insecure=True).upload()
+
+    # Preparing updated pack folder
+    e2e_tests_utils.cli(f'mkdir {tmpdir}/Packs/{pack_name}_testsuite')
+
+    print(f'Trying to download the updated playbook from {playbook_name} to {tmpdir}/Packs/{pack_name}_testsuite/Playbooks')
+    Downloader(output=f'{tmpdir}/Packs/{pack_name}_testsuite', input=[playbook_name], insecure=True).download()
+    assert path.exists(f'{tmpdir}/Packs/{pack_name}_testsuite/Playbooks/{playbook_name}.yml')
+
+    print('Generating docs (creating a readme file)'
+          f' for the playbook {tmpdir}/Packs/{pack_name}_testsuite/Playbooks/{playbook_name}.yml'
+          )
+    generate_playbook_doc.generate_playbook_doc(input_path=f'{tmpdir}/Packs/{pack_name}_testsuite/Playbooks/{playbook_name}.yml')
+    assert path.exists(f'{tmpdir}/Packs/{pack_name}_testsuite/Playbooks/{playbook_name}_README.md')
+
+    print(f'Formating playbook {tmpdir}/Packs/{pack_name}_testsuite/Playbooks/{playbook_name}.yml')
+    format_manager(input=f'{tmpdir}/Packs/{pack_name}_testsuite/Playbooks/{playbook_name}.yml', assume_yes=True)
+    print(f'Validating playbook {tmpdir}/Packs/{pack_name}_testsuite/Playbooks/{playbook_name}.yml')
+    ValidateManager(file_path=f'{tmpdir}/Packs/{pack_name}_testsuite/Playbooks/{playbook_name}.yml').run_validation()
+
+    print(f'Uploading updated playbook {tmpdir}/Packs/{pack_name}_testsuite/Playbooks/{playbook_name}.yml')
+    Uploader(input=f'{tmpdir}/Packs/{pack_name}/Playbooks/{playbook_name}.yml', insecure=True).upload()
+
+
+def test_e2e_demisto_sdk_flow_playbook_client(tmpdir, insecure: bool = True):
+    unique_id = random.randint(1, 1000)
+    pack_name = 'foo_' + str(unique_id)
+    playbook_name = 'pb_' + str(unique_id)
+
+    demisto_client = e2e_tests_utils.connect_to_server(insecure=insecure)
+    body = [
+        {
+            "name": playbook_name,
+            "propagationLabels": [
+                "all"
+            ],
+            "tasks": {
+                "0": {
+                    "id": "0",
+                    "unqiueId": "0",
+                    "type": "start",
+                    "nextTasks": None,
+                    "task": {}
+                }
+            }
+        }
+    ]
+
+    header_params = {}
+    header_params['Accept'] = 'application/json'  # noqa: E501
+    header_params['Accept-Encoding'] = 'gzip, deflate, br'
+    header_params['Content-Type'] = 'application/json'
```

Comment:
we can do `header_params = { a:a_value , b:b_value }` instead

Reply:
[Done](https://github.com/demisto/demisto-sdk/pull/2269/commits/28de6af8cb2b1b6574220d56f323fe802c3e34b4).